installer/Product.wxs

```diff
@@ -0,0 +1,507 @@
+
+		<!--Install NetEbpfExt Driver-->
+		<!--A kernel driver service is not currently supported by the Windows Installer (Wix v3 - https://wixtoolset.org/docs/v3/xsd/wix/serviceinstall/).-->
+		<ComponentGroup Id="NetEbpfExt_Driver" Directory="dir_drivers">
+			<Component Id="NetEbpfExt.inf" Guid="{F1E813EE-665E-488C-84AA-0F47011C12F0}">
+				<File Id="NetEbpfExt.inf" Name="NetEbpfExt.inf" Source="$(var.netebpfext.TargetDir)NetEbpfExt.inf" KeyPath="yes" />
+			</Component>
+			<Component Id="NetEbpfExt.sys" Guid="{3ADAA711-7F62-472A-A60D-589D543ED507}">
+				<File Id="NetEbpfExt.sys" Name="NetEbpfExt.sys" Source="$(var.netebpfext.TargetPath)" KeyPath="yes" />
+			</Component>
+			<Component Id="NetEbpfExt.pdb" Guid="{3B35751C-693A-47C0-A258-97741D8B72CC}">
+				<File Id="NetEbpfExt.pdb" Name="NetEbpfExt.pdb" Source="$(var.netebpfext.TargetDir)NetEbpfExt.pdb" KeyPath="yes" />
+			</Component>
+		</ComponentGroup>
+		<SetProperty Id="NetEbpfExt_Driver_install" Value='"sc.exe" create NetEbpfExt.sys type=kernel start=demand binpath="[#NetEbpfExt.sys]"' Before="NetEbpfExt_Driver_install" Sequence="execute"/>
+		<CustomAction Id="NetEbpfExt_Driver_install" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Return="check" Impersonate="no"/>
+		<Property Id="NetEbpfExt_Driver_start" Value='"net.exe" start NetEbpfExt.sys' />
+		<CustomAction Id="NetEbpfExt_Driver_start" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Return="check" Impersonate="no"/>
+		<Property Id="NetEbpfExt_Driver_stop" Value='"net.exe" stop NetEbpfExt.sys' />
+		<CustomAction Id="NetEbpfExt_Driver_stop" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Return="ignore" Impersonate="no"/>
+		<Property Id="NetEbpfExt_Driver_uninstall" Value='"sc.exe" delete NetEbpfExt.sys' />
+		<CustomAction Id="NetEbpfExt_Driver_uninstall" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Return="ignore" Impersonate="no"/>
+		<Property Id="NetEbpfExt_Driver_stop_rollback" Value='"net.exe" stop NetEbpfExt.sys' />
+		<CustomAction Id="NetEbpfExt_Driver_stop_rollback" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="rollback" Return="ignore" Impersonate="no"/>
+		<Property Id="NetEbpfExt_Driver_uninstall_rollback" Value='"sc.exe" delete NetEbpfExt.sys' />
+		<CustomAction Id="NetEbpfExt_Driver_uninstall_rollback" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="rollback" Return="ignore" Impersonate="no"/>
+	</Fragment>
+
+	<!-- Define the product's <Development> components -->
+	<Fragment>
+		<ComponentGroup Id="eBPF_Development_inlude" Directory="dir_include">
+			<Component Id="BPF_ENDIAN.H" DiskId="1" Guid="59B40FBC-2842-449B-9101-CFBCCF785335">
+				<File Id="BPF_ENDIAN.H" Name="bpf_endian.h" Source="..\include\bpf_endian.h" />
+			</Component>
+			<Component Id="BPF_HELPER_DEFS.H" DiskId="1" Guid="EA36437E-777C-4FFE-ADF5-D806BBDB46FE">
+				<File Id="BPF_HELPER_DEFS.H" Name="bpf_helper_defs.h" Source="..\include\bpf_helper_defs.h" />
```

Comment:
If we add another file to the include directory, it looks like we need to update this file, correct?  Any way to make this automatic in the future?

Reply:
Unfortunately not (and understandably): the installer's paradigm is to *intentionally install and track components*, for reliable handling during installation and uninstallation.
What components are released with a production MSI has always to be under 100% control.